<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-VKM30B9" failures="1" tests="3" name="logics" time="126.31" errors="0" timestamp="2023-03-06T07:18:23 IST">
  <testcase classname="test.RahulShettyTest" name="broken" time="5.157">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	https://rahulshettyacademy.com/brokenlink is broken expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
https://rahulshettyacademy.com/brokenlink is broken expected [true] but found [false]
at pageObjects.PracticePageObjects.brokenLinks(PracticePageObjects.java:96)
at test.RahulShettyTest.broken(RahulShettyTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- broken -->
  <testcase classname="test.RahulShettyTest" name="openLinks" time="41.818"/>
  <testcase classname="test.RahulShettyTest" name="webdriverScope" time="0.569"/>
</testsuite> <!-- logics -->
